_posts/phuong-my_do/2017-04-28-Business_analyst.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Tea Chat with a Business Analyst (BA)"
+breadcrumb: true
+author: phuong-my_do
+date: 2017-06-19
+categories: phuong-my_do
+tags:
+    - business analyst
+teaser:
+    info: A Business Analyst working in Bioinformatics in a Software Engineering Team, what is that?
+---
+
+__What is a Business Analyst in the Cancer Genomics and Bioinformatics world?__
+
+If asking this question to employees in a Cancer Research Institute, chances are 90% of respondents would have no idea; 10% may have some ideas... and we don’t want to know how many of the 10% would be able to describe a Business Analyst job. :)
+
+Here’s Brian, the Business Analyst.
+
+<figure>
+    <img src="{{site.urlimg}}phuong-my_do/brian-the-business-analyst.png" />
+</figure>
+
+*<https://rthewitt.com/2013/04/11/brian-the-business-analyst/>*
+
+This article aims at describing what Brian does in his company illustrated by personal experience, with more examples focused on Cancer Genomics and Bioinformatics. 
+However keep in mind that Brian could do the exact same job for an insurance or sportswear company or any industries with Users who need to use an application to perform their job.
+
+# Who are the players in an IT project and interactions with Brian?
+
+### The Business Stakeholders
+They are the product owner (in our context the principal investigator), and the subject matter experts or super-users (the bioinformaticians and lead scientists).
+
+This is the starting point of all projects: what do the stakeholders want? The higher the hierarchy the higher level the requirements. The stakeholder’s role is to give ideas of what they want but Brian is here to dig into the topic and provide more details before any discussion can happen with the software engineering team. 
+
+### The Project Manager (PM)
+Brian is responsible for ensuring requirements get done. The PM ensures the requirements get developed on a schedule.
```

Comment:
The BA does the specifications of the requirements but the requirements themselves must be implemented and the BA makes sure they are implemented according to what the users wanted, it's sort of referring to the user acceptance testing phase.